cypress/e2e/yves/claim/claim.cy.ts

```diff
@@ -0,0 +1,313 @@
+      claimListPage.openLatestClaimDetailsPage();
+
+      claimDetailPage.getCancelClaimButton().should('not.exist');
+    });
+
+    it('customer with corresponding permission can see claims created by other customers withing the same business unit', (): void => {
+      customerLoginScenario.execute({
+        email: dynamicFixtures.customer.email,
+        password: staticFixtures.defaultPassword,
+        withoutSession: true,
+      });
+
+      claimListPage.visit();
+
+      claimListPage.clickCreateClaimButton();
+
+      claimCreatePage.createClaim(staticFixtures.generalClaim);
+
+      claimDetailPage.assertPageLocation();
+
+      customerLogoutScenario.execute();
+
+      customerLoginScenario.execute({
+        email: dynamicFixtures.customer2.email,
+        password: staticFixtures.defaultPassword,
+        withoutSession: true,
+      });
+
+      claimListPage.visit();
+
+      claimListPage.visit();
+      claimListPage.openLatestClaimDetailsPage();
+
+      claimDetailPage.assertPageLocation();
+    });
+
+    it('customer with corresponding permission can see claims created by other customers withing the same company', (): void => {
+      customerLoginScenario.execute({
+        email: dynamicFixtures.customer.email,
+        password: staticFixtures.defaultPassword,
+        withoutSession: true,
+      });
+
+      claimListPage.visit();
+
+      claimListPage.clickCreateClaimButton();
+
+      claimCreatePage.createClaim(staticFixtures.generalClaim);
+
+      claimDetailPage.assertPageLocation();
+
+      customerLogoutScenario.execute();
+
+      customerLoginScenario.execute({
+        email: dynamicFixtures.customer6.email,
+        password: staticFixtures.defaultPassword,
+        withoutSession: true,
+      });
+
+      claimListPage.visit();
+
+      claimListPage.visit();
+      claimListPage.openLatestClaimDetailsPage();
+
+      claimDetailPage.assertPageLocation();
+    });
+
+    it("customer without corresponding permission shoudn't see claims created by other customers withing the same business unit", (): void => {
+      customerLoginScenario.execute({
+        email: dynamicFixtures.customer3.email,
+        password: staticFixtures.defaultPassword,
+        withoutSession: true,
+      });
+
+      claimListPage.visit();
+
+      claimListPage.clickCreateClaimButton();
+
+      claimCreatePage.createClaim(staticFixtures.generalClaim);
+
+      claimDetailPage.assertPageLocation();
+
+      customerLogoutScenario.execute();
+
+      customerLoginScenario.execute({
+        email: dynamicFixtures.customer4.email,
+        password: staticFixtures.defaultPassword,
+        withoutSession: true,
+      });
+
+      claimListPage.visit();
+      claimListPage.assertPageLocation();
+      claimListPage.assetPageHasNoClaims();
+    });
+
+    it('customer with corresponding permission can see claims created by other customers withing the same business unit', (): void => {
+      customerLoginScenario.execute({
```

Comment:
what this test is about? we already have one called `customer with corresponding permission can see claims created by other customers withing the same business unit`